.github/workflows/build.yml

```diff
@@ -2,7 +2,7 @@ name: Build and Publish
 
 on:
   push:
-    branches: [ "main" ] 
+    branches: [ "task_47" ] 
```

Comment:
shouldnt this be for on push to main? also is there a reason you took out the other python version tests?

Reply:
True.